test-cookie-writer failing on master

The tests at `extensions/amp-analytics/0.1/test/test-cookie-writer.js` are failing and blocking the canary release.

https://travis-ci.org/ampproject/amphtml/jobs/541467573

The fix should cherry picked before the Canary + 1% are shipped and promoted.

```
amp-analytics.cookie-writer value
     
      ● should write cookie with custom expiration (zero value)
        Chrome 75.0.3770 (Linux 0.0.0)
      AssertionError: expected 'aCookie=testValue; path=/; domain=example.com; expires=Mon, 01 Jan 2018 08:00:00 GMT' to equal 'aCookie=testValue; path=/; expires=Mon, 01 Jan 2018 08:00:00 GMT'
          at [...]/amphtml/extensions/amp-analytics/0.1/test/test-cookie-writer.js:377:48

      ● should write cookie with custom expiration (negative value)
        Chrome 75.0.3770 (Linux 0.0.0)
      AssertionError: expected 'aCookie=testValue; path=/; domain=example.com; expires=Mon, 25 Dec 2017 08:00:00 GMT' to equal 'aCookie=testValue; path=/; expires=Mon, 25 Dec 2017 08:00:00 GMT'
          at [...]/amphtml/extensions/amp-analytics/0.1/test/test-cookie-writer.js:398:48

```

--- a/build-system/pr-check/build.js
+++ b/build-system/pr-check/build.js
@@ -62,7 +62,8 @@ function main() {
     if (
       buildTargets.has('RUNTIME') ||
       buildTargets.has('FLAG_CONFIG') ||
-      buildTargets.has('INTEGRATION_TEST')
+      buildTargets.has('INTEGRATION_TEST') ||
+      buildTargets.has('UNIT_TEST')
     ) {
       timedExecOrDie('gulp update-packages');
       timedExecOrDie('gulp build --fortesting');

--- a/build-system/pr-check/dist-bundle-size.js
+++ b/build-system/pr-check/dist-bundle-size.js
@@ -66,7 +66,8 @@ function main() {
       buildTargets.has('FLAG_CONFIG') ||
       buildTargets.has('INTEGRATION_TEST') ||
       buildTargets.has('E2E_TEST') ||
-      buildTargets.has('VISUAL_DIFF')
+      buildTargets.has('VISUAL_DIFF') ||
+      buildTargets.has('UNIT_TEST')
     ) {
       timedExecOrDie('gulp update-packages');
       timedExecOrDie('gulp dist --fortesting');

--- a/extensions/amp-analytics/0.1/test/test-cookie-writer.js
+++ b/extensions/amp-analytics/0.1/test/test-cookie-writer.js
@@ -375,7 +375,8 @@ describes.fakeWin('amp-analytics.cookie-writer value', {amp: true}, env => {
     );
     return cookieWriter.write().then(() => {
       expect(win.document.lastSetCookieRaw).to.equal(
-        'aCookie=testValue; path=/; expires=Mon, 01 Jan 2018 08:00:00 GMT'
+        'aCookie=testValue; path=/; domain=example.com; ' +
+          'expires=Mon, 01 Jan 2018 08:00:00 GMT'
       );
     });
   });
@@ -396,7 +397,8 @@ describes.fakeWin('amp-analytics.cookie-writer value', {amp: true}, env => {
     );
     return cookieWriter.write().then(() => {
       expect(win.document.lastSetCookieRaw).to.equal(
-        'aCookie=testValue; path=/; expires=Mon, 25 Dec 2017 08:00:00 GMT'
+        'aCookie=testValue; path=/; domain=example.com; ' +
+          'expires=Mon, 25 Dec 2017 08:00:00 GMT'
       );
     });
   });

